olive/systems/docker/docker_system.py

```diff
@@ -217,3 +229,11 @@ def _run_container(
     def remove(self):
         self.docker_client.images.remove(self.image.tags[0], force=True)
         logger.info(f"Image {self.image.tags[0]} removed successfully.")
+
+
+def _print_docker_logs(logs, level=logging.INFO):
+    for log in logs:
+        if "stream" in log:
+            logger.log(level, str(log["stream"]).strip())
```

Comment:
I believe raw stream code has ANSI escape codes in it. Can you remove those ANSI escape codes?

Reply:
I don't see the ANSI code in the log. What's kind of logic do you suggest?

Reply:
Can you try to run a docker system example locally with this log?

Reply:
I just see the logging in https://aiinfra.visualstudio.com/PublicPackages/_build/results?buildId=365372&view=logs&j=85be3871-995a-5221-b870-d83ec240e0f0&t=e226bb7e-6fac-5b6d-c5b8-b0e00b847416.

Could you please help point me the location?